src/common/metrics.py

```diff
@@ -98,6 +98,30 @@ def log_figure(self, figure, artifact_file):
         except mlflow.exceptions.MlflowException:
             self._logger.critical(f"Could not log figure using MLFLOW due to exception:\n{traceback.format_exc()}")
 
+    def log_artifact(self, local_path, artifact_path=None):
+        """Logs an artifact
+        
+        Args:
```

Comment:
I'm just reproducing all the regular mlflow methods. mlflow has both, and I believe the idea here is that you can choose to pick only one artefact from its path, or an entire dir.

It could be worth combining both in our code, but we could lose the feature parity with mlflow.

Does that sound right?

Reply:
Sounds fair.